tests/ttnn/integration_tests/resnet/test_ttnn_functional_resnet50.py

```diff
@@ -0,0 +1,83 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import ttnn
+
+from loguru import logger
+
+from models.demos.ttnn_resnet.tests.resnet50_test_infra import create_test_infra
+from models.utility_functions import (
+    is_wormhole_b0,
+    enable_memory_reports,
+    skip_for_wormhole_b0,
+    skip_for_grayskull,
+)
+
+
+@skip_for_grayskull("This is Blackhole specific test")
+@skip_for_wormhole_b0("This is Blackhole specific test")
```

Comment:
Can we add a `@run_for_blackhole()` instead of putting individual skips on other arches? Can add the definition in `models/utility_functions.py` similar to run_for_wormhole_b0
Ex
```
def run_for_wormhole_b0(reason_str="only runs for Wormhole B0"):
    return pytest.mark.skipif(not is_wormhole_b0(), reason=reason_str)
```